db/migrate/20240915055133_add_index_for_stock_price_tickers.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexForStockPriceTickers < ActiveRecord::Migration[8.0]
+  def change
+    add_index :stock_prices, :ticker
```

Comment:
Heads up — this migration will need to be ran after deploying. We're already fetching by ticker, and we're leaning into that even more in this PR. This'll make the queries quicker. Not that they'd absolutely need it now, but at some point they will.